com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -78,6 +79,11 @@ public static function getDepartmentsList($contractID)  {
               INNER JOIN civicrm_option_group cog
                 ON cov.option_group_id = cog.id
               WHERE hrjr.job_contract_id = %1
+                AND hrjr.start_date <= '$today'
+                AND (
+                  hrjr.end_date = 0
```

Comment:
anyway don't worry , I believe job roles need data structure review ( though it is minor review since it is not complex like job contract extension)